libs/model/src/models/associations.ts

```diff
@@ -89,7 +89,13 @@ export const buildAssociations = (db: DB) => {
     asMany: 'threads',
     onUpdate: 'CASCADE',
     onDelete: 'SET NULL',
-  }).withMany(db.ContestTopic, { asMany: 'contest_topics' });
+  })
+    .withMany(db.ContestTopic, { asMany: 'contest_topics' })
+    .withMany(db.GroupPermission, {
```

Comment:
you can probably do the same with a ManyToMany relationship

Reply:
sure, plz lmk if i should update it or keep this one.

Reply:
no need, both work